### PR TITLE
Snapshots folder android #1666

### LINKED ISSUE #1666: Where can I access the background app preview screenshot in Android?
For iOS, you can visit the following directory and download the screenshots:

`/var/mobile/Containers/Data/Application/$APP_ID/Library/Caches/Snapshots/`

This information is not provided for Android. Is it even possible for an attacker with only file system access to download these screenshots?

### PR DESCRIPTION
**Pull request by legik**
Thank you for submitting a Pull Request to the Mobile Security Testing Guide. Please make sure that:

- [x] Your contribution is written in the 2nd person (e.g. you)
- [ ] Your contribution is written in an active present form for as much as possible.
- [x] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [x] Your contribution has proper formatted markdown and/or code
- [x] Any references to website have been formatted as [TEXT](URL “NAME”)
- [x] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

If your PR is related to an issue. Please end your PR test with the following line:
This PR closes #1666.

Not sure about grammar, check please.

### CODE REVIEW

File: Document/0x05d-Testing-Data-Storage.md [hunk 1]
```diff
@@ -879,6 +879,10 @@ While black-box testing the app, navigate to any screen that contains sensitive
 |---|---|
 | <img src="Images/Chapters/0x05d/1.png" width="500px"/> | <img src="Images/Chapters/0x05d/2.png" width="500px"/> |
 
+On device that supports [`file-based encryption(FBE)`](https://source.android.com/security/encryption/file-based "FBE") snapshots are stored in the `/data/system_ce/<USER_ID>/<IMAGE_FOLDER_NAME>` folder. The name of `<IMAGE_FOLDER_NAME>` depends on vendor but most common names are `snapshots` and `recent_images`. If device is not supporting FBE the `/data/system/<IMAGE_FOLDER_NAME>` folder is used.
```
Comment (cpholguera) [inline Document/0x05d-Testing-Data-Storage.md:882]:
```suggestion
On devices supporting [file-based encryption (FBE)](https://source.android.com/security/encryption/file-based "FBE"), snapshots are stored in the `/data/system_ce/<USER_ID>/<IMAGE_FOLDER_NAME>` folder. `<IMAGE_FOLDER_NAME>` depends on the vendor but most common names are `snapshots` and `recent_images`. If the device doesn't support FBE, the `/data/system/<IMAGE_FOLDER_NAME>` folder is used.
```

File: Document/0x05d-Testing-Data-Storage.md [hunk 2]
```diff
@@ -879,6 +879,10 @@ While black-box testing the app, navigate to any screen that contains sensitive
...
+
+Accessing these folders and snapshots require root.
```
Comment (cpholguera) [inline Document/0x05d-Testing-Data-Storage.md:884]:
```suggestion
> Accessing these folders and the snapshots requires root.
```

File: Document/0x05d-Testing-Data-Storage.md (hunk 1 above)
Comment (cpholguera) [reply Document/0x05d-Testing-Data-Storage.md:882]: is this folder also called  `snapshots` or `recent_images`in the second case (no FBE)?

Comment (legik) [reply Document/0x05d-Testing-Data-Storage.md:882]: I guess it is also depends on vendor. 
I saw only variant with ```/data/system/recent_images``` and also this commit say the same [https://github.com/aosp-mirror/platform_frameworks_base/commit/ef73ee1dd98acfc4a19561367cfc3e4d8bbe06ea](https://github.com/aosp-mirror/platform_frameworks_base/commit/ef73ee1dd98acfc4a19561367cfc3e4d8bbe06ea)

Comment (legik) [reply Document/0x05d-Testing-Data-Storage.md]: @cpholguera are you OK if we will use both folder names as an example in case without FBE?

**Pull request merged**